homeassistant/components/lutron/__init__.py

```diff
@@ -228,7 +166,7 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
                     )
                     entry_data.scenes.append((area.name, keypad, button, led))
 
-                entry_data.buttons.append(LutronButton(hass, area.name, keypad, button))
+                entry_data.buttons.append((area.name, keypad, button))
         if area.occupancy_group is not None:
```

Comment:
This will exclude the useless buttons.
```suggestion
                if button.button_type:
                    entry_data.buttons.append((area.name, keypad, button))
        if area.occupancy_group is not None:
```